aiohttp/connector.py

```diff
@@ -1140,11 +1140,9 @@ async def _wrap_create_connection(
                     socket_factory=self._socket_factory,
                 )
                 # Add ssl_shutdown_timeout for Python 3.11+ when SSL is used
-                if (
-                    sys.version_info >= (3, 11)
-                    and self._ssl_shutdown_timeout is not None
-                    and "ssl" in kwargs
-                    and kwargs["ssl"]
+                if self._ssl_shutdown_timeout is not None and sys.version_info >= (
+                    3,
+                    11,
```

Comment:
Ugh, that's not going to match my grep searching when dropping version support. Hopefully it'll still be caught by pyupgrade/ruff..

Reply:
Ah, got formatted back again in the final version, so not an issue.